htdocs/compta/facture/class/facture.class.php

```diff
@@ -527,6 +527,11 @@ public function create(User $user, $notrigger = 0, $forceduedate = 0)
 			$this->mode_reglement_id = GETPOST('mode_reglement_id', 'int') > 0 ? ((int) GETPOST('mode_reglement_id', 'int')) : $_facrec->mode_reglement_id;
 			$this->fk_account        = GETPOST('fk_account') > 0 ? ((int) GETPOST('fk_account')) : $_facrec->fk_account;
 
+			// if ref_client was not set by caller, use the facref title
+			if (empty($this->ref_client)) {
+				$this->ref_client =  trim($_facrec->title);
```

Comment:
The ref_client is the reference of the invoice on the client side (so often know by the client). Why storing the label of the recuring invoice inside this field ?